src/components/landing/Hero.js

```diff
@@ -1,16 +1,41 @@
 import Link from "next/link";
+import Image from 'next/image';
 import styles from "./hero.module.css";
 function Hero() {
   return (
-    <section className={styles.heroSection}>
-      <div className={styles.heroWrap}>
-        <h1>Find your people</h1>
-        <p>Find communities that interest you</p>
-        <p>Are you a community manager?</p>
-        <p>Add your community to our directory for users to find and join</p>
-        <Link href="/register" className={styles.register}>
-          Get Started
-        </Link>
+    <section className="bg-white">
+      <div className="flex flex-row items-center gap-20 px-96 py-36 sm:text-left md:text-left ">
```

Comment:
@istoga  kindly reduce the padding >10 in line 7

Reply:
Hey @ibimina thanks for looking over this. To clarify which padding should I reduce? There are several on line 7. px or py? Thanks.

Reply:
I made some changes for responsiveness, this may have already taken care of your suggesstion.